packages/addons/tools/mono/package.mk

```diff
@@ -67,8 +67,8 @@ makeinstall_target() {
 addon() {
   mkdir -p "$ADDON_BUILD/$PKG_ADDON_ID"  
 
-  make -C "$PKG_BUILD/.$HOST_NAME" install DESTDIR="$ROOT/$ADDON_BUILD/$PKG_ADDON_ID"
-  make -C "$PKG_BUILD/.$TARGET_NAME" install DESTDIR="$ROOT/$ADDON_BUILD/$PKG_ADDON_ID"
+  make -C "$PKG_BUILD/.$HOST_NAME" install DESTDIR="$ROOT/$ADDON_BUILD/$PKG_ADDON_ID" 1> /dev/null
```

Comment:
also, you should never install directly to $ADDON_BUILD, do it somewhere in $PKG_BUILD (.install_pkg that's default for "make install" is also fine). and then use addon() to copy what you need.